brian2cuda/tests/test_gpu_detection.py

```diff
@@ -88,5 +88,5 @@ def test_no_gpu_detection_preference():
     # Test that disabling gpu detection works when setting gpu_id and compute_capability
     prefs.brian2cuda.detect_gpus = False
     prefs.brian2cuda.gpu_id = 0
```

Comment:
We missed these preferences somewhere. I think they were in `cuda_prefs` before. These should both go into `prefs.devices.cuda_standalone.cuda_backend` (since they are related to GPU configuration).